tests/unit/src/test/scala/tests/StacktraceParseSuite.scala

```diff
@@ -0,0 +1,62 @@
+package tests
+
+import scala.meta.internal.metals.StacktraceAnalyzer
+
+class StacktraceParseSuite extends BaseSuite {
+
+  test("convert-stacktrace-line-to-symbol") {
```

Comment:
Maybe move test inside testConversion and make it all separate tests?

Reply:
changed